epigraphhub/data/ggtrends.py

```diff
@@ -0,0 +1,86 @@
+from pytrends.request import TrendReq
+
+
+def _get_connection(language="en-US", timezone=360, **kwargs):
+    return TrendReq(hl=language, tz=timezone)
+
+
+def _build_payload(keywords: list[str], **kwargs) -> object:
```

Comment:
That would require a new import.

Reply:
yep ... but as any object could be recognized as `object` .. it has the same meaning as `Any`, so it is very generic.
as pandas is a direct dependency it would be much better to have it specified there.

I would import everything needed even if it is just for the type annotation.
But, Iif you don't want that, I guess you can use a string to define the annotation and import the package just when the type checking is executed: https://mypy.readthedocs.io/en/stable/runtime_troubles.html#import-cycles

Reply:
fixed